src/ensembl/src/shared/data/pluralisedNouns.ts

```diff
@@ -0,0 +1,9 @@
+type PluralDictionary = {
+  [name: string]: string;
+};
+
+const pluralDictionary: PluralDictionary = {
+  species: 'species'
+};
+
+export default pluralDictionary;
```

Comment:
I think there will be so few exceptions to the general rule of pluralisation that the dictionary can safely live in the same file as the pluralise function.

Reply:
✔️